src/View.php

```diff
@@ -294,8 +306,6 @@ public function addClass($class)
      * Remove one or several CSS classes from the element.
      *
      * @param string|array $class CSS class name or array of class names
```

Comment:
only string is treated correctly (see explode). Or it should be like this in code:
```
if (is_string($remove_class)) {
    $remove_class = explode(' ', $remove_class);
}
```